tests/Service/NphProgramSummaryServiceTest.php

```diff
@@ -87,4 +87,25 @@ public function testCombineProgramAndOrderSummary()
             $this->assertSame($timepointInfo['timePointDisplayName'], $this->module1data['timePoints'][$timepoint]);
         }
     }
+
+    public function testgetSampleStatusCounts()
```

Comment:
Could you change this method name to camel case `testGetSampleStatusCounts` and also add a return type?